src/components/common/TmBalance.vue

```diff
@@ -142,6 +149,16 @@ export default {
       ]
     }
   },
+  watch: {
+    balancesWithFiat: function() {
+      this.convertedBalances = this.balancesWithFiat.map(
```

Comment:
have you considered making this a computed instead?

Reply:
the issue here is that I cannot query `balancesWithFiat` at the beginning, when the component loads, since I lack of the fiatCurrency parameter. That is why I went the watch way.